pandas/tests/io/test_common.py

```diff
@@ -642,6 +643,16 @@ def close(self):
                 handles.created_handles.append(TestError())
 
 
+@td.skip_if_no("fsspec")
+def test_read_csv_chained_url_no_error():
+    tar_file = "pandas/tests/io/data/tar/test-csv.tar"
+    try:
+        pd.read_csv(f"tar://test.csv::file://{tar_file}", compression=None)
+        pd.read_csv(f"tar://test.csv::file://{tar_file}", compression="infer")
+    except Exception as e:
+        pytest.fail(e)
```

Comment:
What's the benefit of using pytest.fail?

Reply:
I was initially thinking that I need to use somekind of pytest function to validate. But it seems like it wouldn't be the case for this